include/rocksdb/utilities/transaction.h

```diff
@@ -234,27 +235,31 @@ class Transaction {
   virtual Status GetForUpdate(const ReadOptions& options,
                               ColumnFamilyHandle* column_family,
                               const Slice& key, std::string* value,
-                              bool exclusive = true) = 0;
+                              bool exclusive = true,
+                              bool skip_validate = false) = 0;
 
   // An overload of the above method that receives a PinnableSlice
   // For backward compatibility a default implementation is provided
   virtual Status GetForUpdate(const ReadOptions& options,
                               ColumnFamilyHandle* column_family,
                               const Slice& key, PinnableSlice* pinnable_val,
-                              bool exclusive = true) {
+                              bool exclusive = true,
+                              bool skip_validate = false) {
```

Comment:
Callaghan had a suggestion which might loosely apply here: avoid argument/option name which contains negative in it, because if it is false, it is double negative and makes it difficulty for users to tune. "Skip" is kind of such thing. `skip_validate = false` is harder to understand than something like `do_validation = false`.